### ISSUE TITLE
QUERY_ALL_PACKAGES permission

### ISSUE THREAD

**Issue by ThemeMetro** [mgks/Android-SmartWebView#270]
Hello,

Is there must required QUERY_ALL_PACKAGES permission? and what is use of this in web view app,
Can i disable it if not much required.

Thanks.

**Comment by mgks** [comment]
yeah sure you can disable it, i might even remove it soon from project. all it is required for is to check which android app store is available on device, something we can already ignore.

**Comment by ThemeMetro** [comment]
> yeah sure you can disable it, i might even remove it soon from project. all it is required for is to check which android app store is available on device, something we can already ignore.

Thank you very much
